lms/static/coffee/spec/instructor_dashboard/student_admin_spec.coffee

```diff
@@ -0,0 +1,166 @@
+describe 'StudentAdmin', ->
```

Comment:
@smagoun @chrisndodge -- here's another example of pre-existing CoffeeScript tests, just FYI